lib/Field.js

```diff
@@ -640,8 +641,86 @@ class Field {
 
     return this.default;
   }
+
+  _objectsToExpressions(option, args) {
+    const expressions = [];
+
+    args.forEach(arg => {
+      if (
+        option !== 'and' && // `and` already supports objects
+        option !== 'not' && // `not` already supports objects
+        option !== 'or' && // `or` already supports objects
+        isObject(arg) &&
+        !Array.isArray(arg)
+      ) {
+        Object.entries(arg).forEach(([field, value]) => {
+          if (!field.startsWith('_$_')) {
+            expressions.push({ [`_$_${option}`]: [field, value] });
+          }
+        });
+      }
+    });
+
+    return expressions;
+  }
+
+  addOption(option, args) {
```

Comment:
Let's also make this method "private" i.e. `_addOption`

Reply:
Isn't the underscore convention communicating a false sense of private-ness to developers?

Reply:
You're right, it's a false sense of security, but it does at least communicate the intent. We can leave this one be though, I think the reason I didn't make it "private" in Where.js was that it's used outside the class. Resolving :)